ComplianceAsCode/content
# Fix #11895 issue

#### Description:

This PR fixes wrong check on /sbin/audispd on Debian >= 11 and Ubuntu >= 22.04.

#### Rationale:

- Fixes #11895

#### Review Hints:

Build the product:

```
./build_product ubuntu2204
```

To test these changes with Ansible:

```
ansible-playbook build/ansible/ubuntu2204-playbook-cis_level1_server.yml --tags "UBTU-22-651030"
```

To test changes with bash, run the remediation sections:  `xccdf_org.ssgproject.content_rule_aide_build_database`

Expected output:

```
Title   Build and Test AIDE Database
Rule    xccdf_org.ssgproject.content_rule_aide_build_database
Result  pass
```
